src/api/liskApi.js

```diff
@@ -40,6 +40,7 @@
  * @main lisk
  */
 import privateApi from './privateApi';
+import utils from './utils';
```

Comment:
Since we're here anyway, it would be nice to destructure the functions out of `utils` directly here.